.github/workflows/tests.yaml

```
@@ -10,7 +10,7 @@ jobs:
       matrix:
         # os: ["windows-latest", "ubuntu-latest", "macos-latest"]
         os: ["ubuntu-latest"]
-        python-version: ["3.7", "3.8", "3.9"]
+        python-version: ["3.8", "3.9", "3.10"]
```

I think we can do this too 

```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

---

Not yet. We need numba/numba#8304

---

Ah, I see. In that case it seems strange to me that we're adding a `ci/environment-3.11.yaml` file.

---

Good point. I can remove it from this PR.